client/test/progression.go

```diff
@@ -34,27 +34,33 @@ type Watcher struct {
 	log.Logger
 	registered chan *channel.RegisteredEvent
 	progressed chan *channel.ProgressedEvent
+	concluded  chan *channel.ConcludedEvent
 }
 
 func makeWatcher(log log.Logger) Watcher {
 	return Watcher{
 		Logger:     log,
 		registered: make(chan *channel.RegisteredEvent),
 		progressed: make(chan *channel.ProgressedEvent),
+		concluded:  make(chan *channel.ConcludedEvent),
```

Comment:
Wouldn't it be better to give those channels some buffer capacity? AFAIK you never read from `concluded`, yet you feed events into it.

Reply:
Thanks for catching this one. This was a left over from some tests I did while trying to improve stability. Indeed wasn't needed in the end.